recipes/pyfesom2/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "pyfesom2" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/FESOM/pyfesom2/archive/{{ version }}.tar.gz
+  sha256: 90363de4f6b78f9eec72ac1f030ae2561062cd9c05a7ce6736f886fd1cf6c347
+
+build:
+  number: 0
+  skip: true  # [win]
+  entry_points:
+    - pfplot=pyfesom2.pfplot:pfplot
+    - pfinterp=pyfesom2.pfinterp:pfinterp
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - basemap
```

Comment:
I think that all you need under `host` is `python`, `pip` and `pytest-runner`. The rest are only required to run the package.